sloppy-deployment/entrypoint.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash -eu
 
 SLOPPY_OUTPUT_FILE="sloppy.yml"
+SLOPPY_SAFE_OUTPUT="${SLOPPY_SAVE_OUTPUT:-}"
```

Comment:
Why another name than ˋSLOPPY_SAVE_OUTPUT`? Why the typo?

Reply:
Hopefulley done. I wasn't sure of what you meant. Please take a look.